### DISCUSSION TITLE
Can i use Kedro for python webprojects instead of data projects?

### DISCUSSION THREAD

**Discussion by simkimsia** [kedro-org/kedro#1173] [Idea]
I came over from https://github.com/cookiecutter/cookiecutter/issues/794#issuecomment-790819024

I wonder if i can use Kedro's framework to create my own python web project templates?

In particular i want a way to check the cookiecutter variable-inputs for invalid character and enforce the constraints.

**Comment by datajoely** [comment]
Hi @simkimsia I think you technically could - but you would only be using the parts of cookiecutter we use without modification. 

Something like this is probably better suited to you needs: https://fastapi.tiangolo.com/project-generation/

**Reply by limdauto** [reply]
Sorry I clicked `Mark as answer` by mistake. I agree with @datajoely. Your best bet is to use a cookiecutter template dedicated to web development.

**Reply by datajoely** [reply]
I'm marking it!

**Reply by simkimsia** [reply]
sorry folks was busy with work. Yup! I understand thank you 🙏

**Reply by WaylonWalker** [reply]
I could see kedro working well for a very data heavy site, where the primary focus is the data.  Process the data and the final datasets are all html/css/js.  It's not what kedro was designed for though.

**Reply by datajoely** [reply]
Also this plugin by @Galileo-Galilei may enable this sort of this one day :) 
https://github.com/Galileo-Galilei/kedro-serving

**Comment by noklam** [comment]
Leaving a new option here now: https://pypi.org/project/kedro-boot/
